django/conf/global_settings.py

```diff
@@ -521,6 +521,8 @@ def gettext_noop(s):
 
 AUTH_PASSWORD_VALIDATORS = []
 
+AUTH_USERNAME_VALIDATORS = []
```

Comment:
Now, to the functionality, my first thought about this approach is that users will expect this to behave like `AUTH_PASSWORD_VALIDATORS`, which is a list containing a dict for each validator, as here:
https://docs.djangoproject.com/en/3.0/topics/auth/passwords/#enabling-password-validation

Did you consider the value in implementing something similar? You would avoid having to implement a list-like object with __len__ and the like, and avoid asking users to import custom classes in settings.py. For more on that:
https://stackoverflow.com/questions/7382722/initializing-custom-objects-in-django-settings